os-3.10-multus/scripts/provision.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+
+set -ex
+
+# Install epel
+yum -y install epel-release
+
+# Install storage requirements for iscsi and cluster
+yum -y install centos-release-gluster
+yum -y install --nogpgcheck -y glusterfs-fuse
+yum -y install iscsi-initiator-utils
+
+cat >/etc/yum.repos.d/origin-latest.repo <<EOF
+[centos-openshift-origin310]
+name=CentOS OpenShift Origin
+baseurl=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin310/
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS
+
+[centos-openshift-origin-testing310]
+name=CentOS OpenShift Origin Testing
+baseurl=http://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin310/
+enabled=0
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/openshift-ansible-CentOS-SIG-PaaS
+
+[centos-openshift-origin-source310]
+name=CentOS OpenShift Origin Source
+baseurl=http://vault.centos.org/centos/7/paas/Source/openshift-origin310/
+enabled=0
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/openshift-ansible-CentOS-SIG-PaaS
+EOF
+
+# Install OpenShift packages
+yum install -y yum-utils \
+  ansible \
+  wget \
+  git \
+  net-tools \
+  bind-utils \
+  iptables-services \
+  bridge-utils \
+  bash-completion \
+  kexec-tools \
+  sos \
+  psacct \
+  docker
+
+curl https://raw.githubusercontent.com/SchSeba/kubevirt-multus-l2-pxe/master/deployment.yaml --output ./multus.yml
```

Comment:
hm can you take it from official repo?

Reply:
There is no official repo for this.

Here I install 3 daemon sets.
1. multus
2. cni plugin binaries
3. custom l2 cni plugin  

want me to just create the yaml in the script and not download it?

Reply:
do you expect some change under this yaml? if yes, I prefer to have it here, to update it only under one place.

Reply:
I don't expect any changes on this yaml.

Reply:
ok, so leave it how it is, will create an image for you

Reply:
Can you move these yamls into the `/manifests` folder in this repo?